build.sbt

```
@@ -105,6 +107,16 @@ val doobie = module("doobie")
   )
   .dependsOn(core % "compile->compile;test->test")
 
+val cassandra = module("cassandra")
+  .settings(
+    libraryDependencies ++= Seq(
+      "com.datastax.oss" % "java-driver-core" % cassandraVersion,
+      "com.dimafeng" %% "testcontainers-scala-scalatest" % testcontainersScalaVersion % Test,
+      "com.dimafeng" %% "testcontainers-scala-cassandra" % testcontainersScalaVersion % Test
+    )
+  )
+  .dependsOn(core)
```

```suggestion
  .dependsOn(core % "compile->compile;test->test")
```

should help the build

---

Thanks, I forgot that -_-